imap_data_access/__init__.py

```diff
@@ -81,3 +81,8 @@
     "<mission>_<instrument>_<datalevel>_<descriptor>_"
     "<startdate>(-<repointing>)_<version>.<extension>"
 )
+
+ANCILLARY_FILENAME_CONVENTION = (
+    "<mission>_<instrument>_<ancillary_name>_"
```

Comment:
Nice, minor nitpick would be to replace "ancillary_name" with "descriptor" just to make it a little more clear.